mathics/builtin/__init__.py

```diff
@@ -1,33 +1,88 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
+from mathics.settings import ENABLE_FILES_MODULE
+
+# Could this be loaded on the fly?
+module_names = [
+    "algebra",
+    "arithmetic",
+    "assignment",
+    "attributes",
+    "calculus",
+    "combinatorial",
+    "compilation",
+    "comparison",
+    "control",
+    "datentime",
+    "diffeqns",
+    "evaluation",
+    "exptrig",
+    "functional",
+    "graphics",
+    "graphics3d",
+    "image",
+    "inout",
+    "integer",
+    "iohooks",
+    "linalg",
+    "lists",
+    "logic",
+    "manipulate",
+    "quantities",
+    "numbertheory",
+    "numeric",
+    "options",
+    "patterns",
+    "plot",
+    "physchemdata",
+    "randomnumbers",
+    "recurrence",
+    "specialfunctions",
+    "scoping",
+    "strings",
+    "structure",
+    "system",
+    "tensors",
+    "xmlformat",
+    "optimization"
+]
 
-from mathics.builtin import (
-    algebra, arithmetic, assignment, attributes, calculus, combinatorial, compilation,
-    comparison, control, datentime, diffeqns, evaluation, exptrig, functional,
-    graphics, graphics3d,
-    image, inout, integer, iohooks, linalg, lists, logic,
-    manipulate, quantities, numbertheory, numeric, options, patterns,
-    plot, physchemdata, randomnumbers, recurrence, specialfunctions, scoping,
-    strings, structure, system, tensors, xmlformat, optimization)
+if ENABLE_FILES_MODULE:
+    # from mathics.builtin import files, importexport
+    module_names += ["files", "importexport"]
+
+modules = []
+
+for module_name in module_names:
+    exec(f"from mathics.builtin import {module_name}")
```

Comment:
I think importlib is what you want to use here instead of exec

Reply:
I'm not sure how to implement the equivalent of the line 58 with importlib. How would you do that?

Reply:
https://github.com/rocky/python3-trepan/blob/master/trepan/processor/cmdproc.py#L1062